EIPS/eip-1024.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 1024
+title: Add web3.eth.encrypt and web3.eth.decrypt functions
+author: Tope Alabi <alabitemitope@gmail.com>
+status: Draft
+type: Interface Track
+created: 2018-05-14
+---
```

Comment:
This seems to be an extension of the RPC protocol, so a `requires: 1474` field would be appropriate.